### PR TITLE
#1245 Add new containerType for Tearsheet in common properties

### LINKED ISSUE #1245: Add support for a new `containerType` for tearsheet in common-properties
ref: 3393

### PR DESCRIPTION
**Pull request by caritaou**
closes #1245 
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by JesusGuerrero** [comment]
@caritaou I did some testing and found the background color on expression is same as the background of tearsheet. For

Screen Shot 2022-11-14 at 11 21 45 AM

For reference, the Tearsheet group for expanding the code editor properly inverts the color.
Screen Shot 2022-11-14 at 11 22 13 AM

**Comment by caritaou** [comment]
@JesusGuerrero Thanks for testing. The tables also needs to be inverted for Tearsheets. I need to verify a few others as well so I will address the colors in a separate issue. 

I fixed the field-picker and long tab title issues that @nmgokhale found for Modal/Tearsheet container. Can you both help review again?

File: canvas_modules/common-canvas/src/common-properties/components/field-picker/field-picker.scss [hunk 1]
```diff
@@ -92,33 +92,16 @@
...
-
-		.properties-fp-button-label {
-			position: relative;
-			top: $spacing-03;
+// This is for field-picker in properties-modal, containerType="Modal"
+.properties-modal, .properties-tearsheet-panel.bx--modal {
+	.properties-fp-table {
+		.properties-fp-top-row {
+			display: flex;
+			justify-content: end;
```
Comment (nmgokhale) [inline canvas_modules/common-canvas/src/common-properties/components/field-picker/field-picker.scss:100]: `npm run build` shows this warning in console -
Screen Shot 2022-11-14 at 4 27 15 PM

**Comment by nmgokhale** [comment]
In fieldpicker table, when number of rows are less after filtering, it shows white background for the table - 
Screen Shot 2022-11-14 at 4 43 24 PM

Someofselect tables don't have borders -
Screen Shot 2022-11-14 at 4 48 33 PM

**Comment by caritaou** [comment]
Thanks @nmgokhale, I fixed the padding for when there is only a single category. The rest will be fixed in a separate issue to invert colors in Tearsheet. I made a note on the items you listed

**Comment by nmgokhale** [comment]
Bottom border for last row in table isn't showing up. It was showing up yesterday - 
Screen Shot 2022-11-15 at 10 04 48 AM

**Comment by caritaou** [comment]
@nmgokhale I still see that in my env
image

**Comment by nmgokhale** [comment]
I see bottom border in firefox and Safari. PR looks good.

**Review by nmgokhale — APPROVED** [review]

**Review by tomlyn — APPROVED** [review]

**Pull request merged**